open-api/rest-catalog-open-api.yaml

```diff
@@ -100,6 +121,16 @@ paths:
 
         Common catalog configuration settings are documented at
         https://iceberg.apache.org/docs/latest/configuration/#catalog-properties
+
+
+        Capabilities are used by the server to signal supported functionality and can be:
+        - tables (default capability in case the `capabilities` property doesn't exist or is empty in the response)
+        - views
+        - remote-signing
```

Comment:
IMHO it feels a bit odd to define a capability here that has no endpoint defined in this doc.

Reply:
this shouldn't be an issue once the S3 Signer spec is merged into this spec (which will be handled as a follow-up), but we still want to list this capability here